<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="Photo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Regression</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>18thCentury</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict IMDB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ac">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Actual_Bar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Predict_Bar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pre">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>18thCentury</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actually IMDB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Classification</string>
   </property>
   <widget class="QProgressBar" name="Actual_CBar1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Actual_CBar2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Actual_CBar3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Actual_CBar4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Pre_CBar1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Pre_CBar2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Pre_CBar3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Pre_CBar4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>18thCentury</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actually Point</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>18thCentury</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict Point </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ac_C">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pre_C">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Feature</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Release_Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age_Certf.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genres</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Year">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Country">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Genres">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Age">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
